server/src/test/java/org/elasticsearch/snapshots/AbstractSnapshotIntegTestCase.java

```diff
@@ -307,4 +310,16 @@ protected String initWithSnapshotVersion(String repoName, Path repoPath, Version
                 StandardOpenOption.TRUNCATE_EXISTING);
         return oldVersionSnapshot;
     }
+
+    protected SnapshotInfo createFullSnapshot(String repoName, String snapshotName) {
```

Comment:
Maybe static?

Reply:
Same explanation as above, the `logger` this thing uses isn't static.